packages/community/src/retrievers/bedrock.ts

```diff
@@ -0,0 +1,160 @@
+import {
+  BedrockAgentRuntimeClient,
+  RetrieveCommand,
+  type BedrockAgentRuntimeClientConfig,
+  type RetrievalFilter,
+  type SearchType,
+} from "@aws-sdk/client-bedrock-agent-runtime";
+
+import { BaseRetriever, Document } from "llamaindex";
```

Comment:
import Document from llamaindex/core/schema and copy BaseRetriever type here for now.

`llamaindex` is the to end user level pacakge and shouldn't be imported from our other pacakge, so please remove `llamaindex` from deps

remind me that we need move `BaseRetriever` into core module

Reply:
Thank you a lot for your review, please proceed with moving `BaseRetriever` into core module. Then, I will update code to import it if you are OKay?
Otherwise, I will copy `BaseRetriever` type here.

Reply:
too late to me, will do tmr